rlcard/games/shortlimitholdem/utils.py

```diff
@@ -0,0 +1,83 @@
+from rlcard.games.limitholdem.utils import Hand
+
+class ShortHand(Hand):
+    ''' Employs an alternative ranking system for a short hand game.
+    Said ranks can be found here: https://www.pokernews.com/poker-rules/six-plus-hold-em.htm
+    '''
+    RANK_TO_STRING = {6: "6", 7: "7", 8: "8", 9: "9", 10: "T",
+                      11: "J", 12: "Q", 13: "K", 14: "A"}
+    RANK_LOOKUP = "A6789TJQKA"
```

Comment:
Is there a reason for the low A? It seems like straight evaluator already checks for an ace and inserts it as a low card.

Reply:
The straight evaluator uses RANK_LOOKUP as apart of its logic. It takes a string of ranks and sees if it exists within RANK_LOOKUP. If the low A was not there, the string "A6789" would not exist and incorrectly return false.